### PR TITLE
PR to fix #1 - write hello world script

### LINKED ISSUE #1: Create Hello World program
Write a program in any language that prints out "Hello World"

### PR DESCRIPTION
**Pull request by arif-syr**
Fixes #1

### CODE REVIEW

File: hello_world.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+if __name__ == "__main__":
+    print("Hello, world!")
```
Comment (zmehta2) [inline hello_world.py:2]: Print Hello **W**orld

**Comment by zmehta2** [comment]
Resolve comments

**Review by zmehta2 — CHANGES_REQUESTED** [review]

File: hello_world.py (hunk 1 above)
Comment (arif-syr) [reply hello_world.py]: Fixed in latest commit

**Review by zmehta2 — APPROVED** [review]

File: hello_world.py (hunk 1 above)
Comment (zmehta2) [reply hello_world.py]: LGTM

**Pull request merged**